### PR TITLE
Visualization that was missing

### PR DESCRIPTION
**Pull request by rfl3**

### CODE REVIEW

File: src/visualizations/newContributorsPRs.js [hunk 1]
```diff
@@ -5,7 +5,8 @@ import { donutChart } from '../providers/dataVisualizer';
 
 async function numberPrsNewContributorsAccepted(user, repo, size, divId) {
   const prs = await getPrs(user, repo);
-
+  // for (let i = 0; i < prs.length; i += 1) {
```
Comment (MVictorM) [inline src/visualizations/newContributorsPRs.js]: remove

File: src/requestedChanges.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+      } else {
+        data[prs[i].numberOfChanges] = [1, 0];
+      }
+    }
+  }
+
+  const keys = Object.keys(data);
+  console.log(keys);
+  for (let i = 0; i < keys.length; i += 1) {
+    console.log(keys[i]);
```
Comment (MVictorM) [inline src/requestedChanges.js]: remove

File: src/requestedChanges.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+      } else if (data[prs[i].numberOfChanges]) {
+        data[prs[i].numberOfChanges][0] += 1;
+      } else {
+        data[prs[i].numberOfChanges] = [1, 0];
+      }
+    }
+  }
+
+  const keys = Object.keys(data);
+  console.log(keys);
```
Comment (MVictorM) [inline src/requestedChanges.js]: remove

File: src/requestedChanges.js [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+  for (let i = 0; i < keys.length; i += 1) {
```
Comment (MVictorM) [inline src/requestedChanges.js]: remove

File: src/visualizations/newContributorsPRs.js (hunk 1 above)
Comment (rfl3) [reply src/visualizations/newContributorsPRs.js]: done

File: src/requestedChanges.js (hunk 2 above)
Comment (rfl3) [reply src/requestedChanges.js]: done

**Pull request merged**